4-AdvancedGrants/2-call-api-api-ca/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,135 @@
+{
+    "Sample": {
+      "Title": "Handling Conditional Access challenges in an Azure AD protected Node.js web API calling another protected Node.js web API on behalf of a user",
+      "Level": 300,
+      "Client": "JavaScript SPA",
+      "Service": "Node.js web API",
+      "RepositoryUrl": "ms-identity-javascript-tutorial",
+      "Endpoint": "AAD v2.0"
+    },
+    "AADApps": [
+      {
+        "Id": "DownstreamAPI",
+        "Name": "ms-identity-javascript-tutorial-c4s2-downstream",
+        "Kind": "WebApi",
+        "HomePage": "http://localhost:7000/api",
+        "Audience": "AzureADMyOrg",
+          "ManualSteps": [
+            {
+              "Comment": "Navigate to the Manifest and set 'accessTokenAcceptedVersion' to '2' instead of 'null'"
```

Comment:
>Navigate to the Manifest and set 'accessTokenAcceptedVersion' to '2' instead of 'null' [](start = 26, length = 86)

add a new step "Also create a new conditional access policy as described in the sample's readme"